Library/Homebrew/dev-cmd/boneyard-formula-pr.rb

```diff
@@ -0,0 +1,130 @@
+      puts "cp #{formula_path.relative_path_from(formula.tap.path)} #{boneyard_tap.formula_dir}"
+      puts "git rm #{formula_path.relative_path_from(formula.tap.path)}"
+      unless File.exist? formula.tap.path/"tap_migrations.json"
+        puts "Creating tap_migrations.json for #{formula.tap.name}"
+        puts "git add #{formula.tap.path}/tap_migrations.json"
+      end
+      puts "Loading tap_migrations.json"
+      puts "Adding #{formula.name} to tap_migrations.json"
+    else
+      safe_system HOMEBREW_BREW_FILE, "update"
+      safe_system HOMEBREW_BREW_FILE, "tap", boneyard_tap.name
+      cd formula.tap.path
+      cp formula_path.relative_path_from(formula.tap.path), boneyard_tap.formula_dir
+      safe_system "git", "rm", formula_path.relative_path_from(formula.tap.path)
+      unless File.exist? formula.tap.path/"tap_migrations.json"
+        (formula.tap.path/"tap_migrations.json").write <<-EOS.undent
+          {
+          }
+        EOS
+        safe_system "git", "add", formula.tap.path/"tap_migrations.json"
+      end
+      tap_migrations = Utils::JSON.load(File.read(formula.tap.path/"tap_migrations.json"))
+      tap_migrations[formula.name] = boneyard_tap.name
+      tap_migrations = tap_migrations.sort.inject({}) { |a, e| a.merge!(e[0] => e[1]) }
+      (formula.tap.path/"tap_migrations.json").atomic_write(JSON.pretty_generate(tap_migrations) + "\n")
+    end
+    unless Formula["hub"].any_version_installed?
+      if ARGV.dry_run?
+        puts "brew install hub"
+      else
+        safe_system HOMEBREW_BREW_FILE, "install", "hub"
+      end
+    end
+    branch = "#{formula.name}-boneyard"
+    puts "cd #{formula.tap.formula_dir}" if ARGV.dry_run?
+    cd formula.tap.formula_dir
+    if ARGV.dry_run?
+      puts "git checkout -b #{branch} origin/master"
+      puts "git commit --no-edit --verbose --message=\"#{formula.name}: migrate to boneyard\" -- #{formula_path} #{formula.tap.path}/tap_migrations.json"
+      puts "hub fork --no-remote"
+      puts "hub fork"
+      puts "hub fork (to read $HUB_REMOTE)"
+      puts "git push $HUB_REMOTE #{branch}:#{branch}"
+      puts "hub pull-request -m $'#{formula.name}: migrate to boneyard\\n\\nCreated with `brew boneyard-formula-pr`.'"
+    else
+      safe_system "git", "checkout", "-b", branch, "origin/master"
+      safe_system "git", "commit", "--no-edit", "--verbose",
+        "--message=#{formula.name}: migrate to boneyard",
+        "--", formula_path, formula.tap.path/"tap_migrations.json"
+      safe_system "hub", "fork", "--no-remote"
+      quiet_system "hub", "fork"
+      remote = Utils.popen_read("hub fork 2>&1")[/fatal: remote (.+) already exists./, 1]
+      odie "cannot get remote from 'hub'!" unless remote
+      safe_system "git", "push", remote, "#{branch}:#{branch}"
+      first_pull_request_url = Utils.popen_read("hub", "pull-request", "-m",
+       "#{formula.name}: migrate to boneyard\n\nCreated with `brew boneyard-formula-pr`.").chomp
```

Comment:
Save the message in a variable and then use it in the `popen_read` call? Also feel free to change `first_pull_request_url` to something nicer, I wasn't really suggesting to use this name literally.